lms/djangoapps/discussion_api/api.py

```diff
@@ -274,6 +277,7 @@ def get_thread_list(
         "last_activity_at".
     order_direction: The direction in which to sort the threads by. The only
         values are "asc" or "desc". The default is "desc".
+    discussion_id: Content-specific discussion topic ID
```

Comment:
@nimisha In that case, the semantic for 'topic_id_list' is same as 'discussion_id' list. Both 'comments_sevice' and 'DiscussionApi' considers inline discussion ids as 'topic_id/commentable_id'
I would suggest to keep using 'topic_id_list' and remove 'discussion_id' to avoid managing two different fields for same purpose.